sites/all/modules/ar/ar_assessments/js/ar_assessments.js

```diff
@@ -405,7 +405,7 @@ Drupal.behaviors.arAssessmentsAssessments = {
       },
 
       defaultRoute: function (actions) {
-        this.navigate('list/1', {trigger: true});
```

Comment:
@amyvbenson Ok on my side, here is the case where I was told to show the list view as default: https://humanitarian.atlassian.net/browse/HRINFO-587 CC @emmajane